staging/src/k8s.io/apiserver/pkg/util/flowcontrol/metrics/metrics.go

```diff
@@ -247,7 +247,7 @@ var (
 			Subsystem:      subsystem,
 			Name:           "request_concurrency_in_use",
 			Help:           "Concurrency (number of seats) occupied by the currently executing (initial stage for a WATCH, any stage otherwise) requests in the API Priority and Fairness subsystem",
-			StabilityLevel: compbasemetrics.ALPHA,
+			StabilityLevel: compbasemetrics.BETA,
```

Comment:
Since we plan to deprecate and remove this metric, and the equal metric `apiserver_flowcontrol_current_executing_seats` is available (since #118960), I think that the latter should become BETA rather than the former.

Reply:
This has been resolved (and GitHub moved the comment to the resolution!)